packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -49,28 +63,23 @@ type Props = {|
 
 export default class ModalPanel extends React.Component<Props> {
     static defaultProps = {
-        showCloseButton: false,
+        closeButtonVisible: true,
         scrollOverflow: true,
-        color: "light",
+        light: true,
     };
 
     maybeRenderCloseButton() {
-        const {showCloseButton, onClose, color, titleBar} = this.props;
+        const {closeButtonVisible, onClose, light} = this.props;
 
-        if (!showCloseButton) {
+        if (!closeButtonVisible) {
             return null;
         }
 
-        // Figure out the background color that'll be behind the close button
-        // If a titlebar is specified then we use that, otherwise we use the
-        // default background color.
-        const topBackgroundColor = (titleBar && titleBar.props.color) || color;
-
         return (
             <MediaLayout styleSheets={styleSheets}>
                 {({styles}) => (
                     <CloseButton
```

Comment:
This could maybe be inlined.  That would avoid an extra `MediaLayout`... also there's less logic now.